proto/desmos/posts/v1beta1/polls.proto

```diff
@@ -0,0 +1,71 @@
+syntax = "proto3";
+package desmos.posts.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/desmos-labs/desmos/x/posts/types";
+
+// PollAnswer contains the data of a single poll answer inserted by the creator
+message PollAnswer {
+  option (gogoproto.equal) = true;
+  option (gogoproto.goproto_stringer) = true;
+
+  string answer_id = 1[
+    (gogoproto.customname) = "ID",
+    (gogoproto.jsontag) = "id",
+    (gogoproto.moretags) = "yaml\"id\""
+  ];
+  string text = 2 [
+    (gogoproto.jsontag) = "text",
```

Comment:
we can remove this if protobuf provide for json tags automatically right?

Reply:
This was added to remove the `omitempty` clause that is automatically added by `gogoproto`. If we removed this, the resulting tag would be: 

```
json:"text,omitempty"
```

Instead, to make it the same as other fields, I've left it to make it 

```
json:"text"
```